legal-api/src/legal_api/utils/legislation_datetime.py

```diff
@@ -54,6 +54,15 @@ def as_utc_timezone(date_time: datetime) -> datetime:
         """Return a datetime adjusted to the GMT timezone (aka UTC)."""
         return date_time.astimezone(pytz.timezone('GMT'))
 
+    @staticmethod
+    def format_as_report_string(date_time: datetime) -> str:
+        """Return a datetime string in this format (eg: `August 5, 2021 at 11:00 am Pacific time`)."""
+        hour = date_time.strftime('%I').lstrip('0')
+        # %p provides locale value: AM, PM (en_US); am, pm (de_DE); So forcing it to be lower in any case
+        am_pm = date_time.strftime('%p').lower()
```

Comment:
%P does it lower case.

Reply:
Yes I agree but the documentation says it depends on locale value

Reply:
The documentation only mentions `%p`. `%P` is only for linux with glibc: https://linux.die.net/man/3/strftime

Reply:
ok. Don't worry about it.